cli/test/e2e/package_install_dry_run_test.go

```diff
@@ -0,0 +1,164 @@
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPackageInstallDryRun(t *testing.T) {
```

Comment:
nit: Should we make it table driven test case?

Reply:
Looks like this is due to the boolean not being appropriately named, fixing that would fix this!

Reply:
There are 3 cases with different flags and this list won't grow, so I think it is fine as is. Why do you think it should be table driven?

Reply:
I missed the different flags. 

Since we were having yaml as input and the output was compared against yaml, table-driven would have been better if we had the same flags...

Reply:
Yep agreed! Input however is flag based here